app/controllers/turfs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TurfsController < ApplicationController
+  def create
+    unless current_user
```

Comment:
place this in a `before_action` to create more clarity as to the intent of the `create` method